static/styles/styles.css

```diff
@@ -0,0 +1,2587 @@
+@charset "UTF-8";
```

Comment:
@noahmanger I think you have to run this to remove the actually file now that its being ignore:

```
git rm --cached <file>
```